src/components/Asset/AssetDetails.tsx

```diff
@@ -19,8 +19,12 @@ export const AssetSymbol: FC<AssetDetailsProps> = ({ address, isLoading }) => {
     address,
   })
   return (
-    <Skeleton isLoading={!address || isLoading || isLoadingToken || !isReady}>
-      {isReady && token?.symbol ? token.symbol : "Loading..."}
+    <Skeleton isLoading={isLoading || isLoadingToken || !isReady}>
+      {isReady
+        ? address
+          ? token?.symbol ?? "Loading..."
+          : "???"
```

Comment:
Because of the way that concentrators need to be requested, it is possible for this to be undefined. It's not something a user will see but while concentrators are being developed this will prevent crashes.